pkgs/development/interpreters/ruby/default.nix

```diff
@@ -134,19 +129,17 @@ let
         '';
 
         configureFlags = ["--enable-shared" "--enable-pthread" "--with-soname=ruby-${version}"]
-          ++ op useRailsExpress "--with-baseruby=${baseruby}/bin/ruby"
+          ++ op useBaseRuby "--with-baseruby=${baseRuby}/bin/ruby"
```

Comment:
I prefer `withFeatureAs useBaseRuby "baseruby" "${baseRuby}/bin/ruby"` here, but that's up to you.

Reply:
that is kind of out of scope here but I added a commit for it - am a bit wary of the new `disable` and `without` flags now but it still seems to work so *shrug* o:

Reply:
I'm sorry, I misread the diff here and thought you were adding a line where none had been before, not just changing an option that already exists.  You're right that it's out of scope.  Feel free to undo the change, or keep it in, as you prefer.

Reply:
May as well while we're here causing massive rebuilds anyway.